_posts/2018-11-16-webhooks-about.markdown

```diff
@@ -0,0 +1,47 @@
+---
+permalink: v1/webhooks/about
+title: Webhooks
+layout: sidebar
+---
+
+<h1 class="block">
+    Webhooks
+</h1>
+
+## About
+Webhooks are a way to get notified when events happen in Gusto so that you do not need to come up with a polling strategy for refreshing data in your app.
+
+Webhooks trigger on certain events and send an HTTP POST payload to the URL that is registered by you for that event. You can limit which events trigger a webhook for your registered URLs.
```

Comment:
Why POST? Aren't some (most) entities being PUT'd (updated)? It might be worth explaining why it's POST in all cases.

Reply:
I believe standard webhooks all use POST, I vaguely remember we discussed this in the design doc.

Reply:
True. Although a developer who is accustomed to a RESTful world will derive meaning from the http verb being used here.

My point was to add a sentence or two to the documentation to explain why it's POST in all cases (`create`, `update`, and `delete`).